cli/command/service/update.go

```diff
@@ -349,6 +349,14 @@ func updateService(flags *pflag.FlagSet, spec *swarm.ServiceSpec) error {
 		cspec.ReadOnly = readOnly
 	}
 
+	if flags.Changed(flagStopSignal) {
+		stopSignal, err := flags.GetString(flagStopSignal)
+		if err != nil {
+			return err
+		}
+		cspec.StopSignal = stopSignal
```

Comment:
I suppose this could use `updateString` for consistency with the code above. It doesn't really matter though.